### ISSUE TITLE
Replying to Community emails should work more like other mailing list systems?

### ISSUE THREAD

**Issue by qqrs** [recursecenter/community#446]
I'm filing this issue because I have, once again, sent an email to hundreds of people when I intended to email the thread originator directly. It's been a couple years since I replied to a Community thread and I'd forgotten it doesn't work like most other mailing lists. Of course it's a little tricky since Community is aiming to be a blend of forum software and a mailing list.

This relates to https://github.com/recursecenter/community/issues/321. Community forges the From header, but uses a Reply-To so replies go back to the Community list.

Google Groups behavior:
```
from:	Original Poster <op@example.com>
to:	Some Mailing List <some-mailing-list@example.com>
``` 
```
"Reply" -> to: op@example.com
"Reply All" -> to: op@example.com, cc: some-email-list@example.com
```

Community email behavior:
```
from:	Original Poster <op@example.com> via mail.community.recurse.com 
reply-to:	general@lists.community.recurse.com
to:	general@lists.community.recurse.com
```
```
"Reply" -> to: general@lists.community.recurse.com
"Reply All" -> to: general@lists.community.recurse.com
```

Possible proposed alternative... What if Community didn't use the Reply-To? Would the user be able to reply to OP only or reply all if they want to send to OP and the list?

(I realize Community isn't under active development, filing this for future consideration)

**Comment by nicholasbs** [comment]
Thanks for the clear report!

+1. The status quo clearly isn't right -- it should probably work how Google Groups works.

I won't promise a specific timeline, but we'll make this change eventually (also happy to take but by no means expect a PR since this is all open source, and that will still be blocked on one of the ~2 people who know this codebase to review/deploy 😄)

**Comment by porterjamesj** [comment]
OK, we tried removing the `reply-to`, and it worked "as expected" in that:

- If you just click "reply" to an email, your reply goes to the original poster and not the list
- If you click "reply all", it goes to the original poster and the list. *However*, this results in your reply appearing in the original poster's email inbox *twice* (one email is the actual email you sent them, and the other is generated by community).

We've rolled back the change for now, since this "double email" behavior seems kind of annoying for the OP.

We want to ask for input first, from @qqrs or anyone else. Does anyone know if this is how other mailing lists (e.g. Google groups) work? And if not, how do they solve this problem? Does this double email behavior seem too annoying? Or is it worth it to have the behavior of clicking reply be more intuitive?

**Comment by qqrs** [comment]
Thanks for taking this on @porterjamesj.

I tested with Google Groups and OP only receives one copy of replies. It comes directly from the reply-er, not via the list (i.e., there's no mailing list and mailed-by header).

[Mailman docs](https://www.gnu.org/software/mailman/mailman-admin/node14.html) suggest this might be done by skipping list recipients if the headers indicate they are already receiving a copy. 
> **Do not send a copy of a member's own post** specifies whether a member posting to the list will get a copy of their own posting.
> **Filter out duplicate messages to list members (if possible)** specifies whether members who are explicitly listed as a recipient of a message (e.g. via the Cc: header) will also get a copy from Mailman.

Think that approach might work?

**Comment by porterjamesj** [comment]
I've now implemented the suggestion @qqrs made above in #456. Replying to emails on Community will now work like Google groups. If you click "Reply", it will go just to the OP. If you click "Reply All", it will go to the entire list, and the OP will only get one copy (directly from you) because Community will skip them when sending to the list because it sees their address is already in the "To:" field.

**Comment by qqrs** [comment]
This is great. Thanks @porterjamesj !
